vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow-client/pom.xml

```diff
@@ -38,7 +38,7 @@
         <!-- we dont put this dependencies in parent poms because they are only
          used in this module just for the compilation Java to JS -->
         <gwt.version>2.9.0</gwt.version>
-        <framework.version>8.27.7</framework.version>
+        <framework.version>8.29.0</framework.version>
```

Comment:
I wasn't able to reproduce the issue in `24.9` branch using Simulator. I wonder if it only occurs on a physical device?

https://github.com/user-attachments/assets/f57f0abb-2225-489f-a383-1df4c17f04a7

Reply:
Can you try it on an iPad? It works fine with iPhone, but the issue is with iPads, actually.

Reply:
Ah, I missed it's only an iPad issue. Was able to repro and confirmed this bump fixes the issue.